example_config.yaml

```diff
@@ -27,16 +27,15 @@ sensors:
     name: "spec_no2_sensor"
     path: "/dev/ttyUSB0"
     # values contain all the different attributes a sensor captures.
-    # The value of each attribute is the number of bytes the device 
-    # will use to send the reading. 
+    # each attribute specifies the data type used to send the reading. 
     values:
-      - temperature: 4
-      - humidity: 0
-      - no2: 2
+      - temperature: int
+      - humidity: short
+      - no2: float
```

Comment:
This should be NO2 to conform to condensed structural formula specification.

Not sure if not changing it would break anything, but, it pays to be consistent